src/main/java/persistence/entity/EntityLoader.java

```diff
@@ -2,33 +2,33 @@
 
 import jdbc.JdbcTemplate;
 import jdbc.RowMapperFactory;
+import persistence.meta.Metamodel;
 import persistence.sql.definition.TableAssociationDefinition;
 import persistence.sql.definition.TableDefinition;
 import persistence.sql.dml.query.SelectQueryBuilder;
 
 public class EntityLoader {
     private final JdbcTemplate jdbcTemplate;
-    private final RowMapperFactory rowMapperFactory;
+    private final Metamodel metamodel;
 
-    public EntityLoader(JdbcTemplate jdbcTemplate) {
+    public EntityLoader(JdbcTemplate jdbcTemplate, Metamodel metamodel) {
         this.jdbcTemplate = jdbcTemplate;
-        this.rowMapperFactory = RowMapperFactory.getInstance();
+        this.metamodel = metamodel;
     }
 
     public <T> T loadEntity(Class<T> entityClass, EntityKey entityKey) {
-        final SelectQueryBuilder queryBuilder = new SelectQueryBuilder(entityKey.entityClass());
-        final TableDefinition tableDefinition = new TableDefinition(entityKey.entityClass());
+        final SelectQueryBuilder queryBuilder = new SelectQueryBuilder(entityKey.entityClass(), metamodel);
+        final TableDefinition tableDefinition = metamodel.getTableDefinition(entityClass);
```

Comment:
```suggestion
        final TableDefinition tableDefinition = metamodel.findTableDefinition(entityClass);
```

저라면, 보통 콜렉션 자료구조에서 사용하는 `꺼내줘(get-)` 보다는 객체에게 데이터를 철저하게 관리하는 느낌을 주기위해 이렇게 할것 같습니다. 
> 수정을 위한 코멘트 아닙니다

Reply:
생각해보니 그런 느낌이 더 많이드는 것 같습니다 ㅎㅎㅎ find로 수정하여 사용하겠습니다.